server/src/internalClusterTest/java/org/elasticsearch/search/aggregations/bucket/RandomSamplerIT.java

```diff
@@ -85,11 +86,13 @@ public void setupSuiteScopeCluster() throws Exception {
         ensureSearchable();
     }
 
-    @AwaitsFix(bugUrl = "https://github.com/elastic/elasticsearch/issues/105839")
     public void testRandomSamplerConsistentSeed() {
         double[] sampleMonotonicValue = new double[1];
         double[] sampleNumericValue = new double[1];
         long[] sampledDocCount = new long[1];
+        // Force merge to ensure segment consistency as any segment merging can change which particular documents
+        // are sampled
+        assertNoFailures(indicesAdmin().prepareForceMerge("idx").setMaxNumSegments(1).get());
```

Comment:
I see that `idx` index is setup and indexed once for all tests in this module. Is this correct?  If so, I have 2 questions:

- Is it being force-merged to 1 segment  having any influence on other tests?
- Does it make sense to move force-merge to `setupSuiteScopeCluster()`?

Reply:
You are correct. I moved the force merge into the setup. That is fine for the other tests as well.